### PR TITLE
feat: Unit tests for serialize_v1 for folders

### PR DESCRIPTION
**Pull request by laurentsimon**
closes https://github.com/google/model-transparency/issues/57

### CODE REVIEW

File: model_signing/serialize_test.py [hunk 1]
```diff
@@ -89,27 +89,6 @@ def signature_path(model: Path) -> Path:
 
 
 class Test_serialize_v0:
-    # symlink in root folder raises ValueError exception.
```
Comment (laurentsimon) [inline model_signing/serialize_test.py:92]: I moved these to be next to the other folder tests.

File: model_signing/serialize_test.py [hunk 2]
```diff
@@ -161,25 +140,6 @@ def test_different_filename(self):
 
         assert (r0 == r1)
 
-    # Folder serialization works.
-    def test_known_folder(self):
```
Comment (laurentsimon) [inline model_signing/serialize_test.py:165]: same comment: just moved next to folder tests

**Review by mihaimaruseac — APPROVED** [review]

File: model_signing/serialize_test.py [hunk 3]
```diff
@@ -621,6 +627,12 @@ def test_different_ignored_paths(self):
         assert (r0 != r2)
         assert (r0 != r3)
         assert (r0 != r4)
+        assert (r1 != r2)
+        assert (r1 != r3)
+        assert (r1 != r4)
+        assert (r2 != r3)
+        assert (r2 != r4)
+        assert (r3 != r4)
```
Comment (mihaimaruseac) [inline model_signing/serialize_test.py:635]: I'm think we could also put all these in a set and check the length of the set to be 5?

**Review by mihaimaruseac — APPROVED** [review]

File: model_signing/serialize_test.py [hunk 4]
```diff
@@ -123,7 +102,7 @@ def test_known_file(self):
 
     # File serialization returns the same results for different chunk sizes.
     def test_file_chuncks(self):
-        file = "model_file"
+        file = "v0_test_file_chuncks"
```
Comment (mihaimaruseac) [inline model_signing/serialize_test.py:105]:
```suggestion
        file = "v0_test_file_chunks"
```

File: model_signing/serialize_test.py [hunk 5]
```diff
@@ -136,7 +115,7 @@ def test_file_chuncks(self):
 
     # File serialization raises error for negative chunk values.
     def test_file_negative_chuncks(self):
-        file = "model_file"
+        file = "v0_test_file_negative_chuncks"
```
Comment (mihaimaruseac) [inline model_signing/serialize_test.py:118]:
```suggestion
        file = "v0_test_file_negative_chunks"
```

We have this typo in several places

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**